src/styles/components/_code-pattern.scss

```diff
@@ -0,0 +1,37 @@
+@import '../settings/colors';
+
+// =============================================================================
+// CODE PATTERN component
+//
+// A component that displays the code and a demo side by side.
+//
+// =============================================================================
+
+.code-pattern {
+  width: 100%;
+
+  .code-pattern__content {
+    display: grid;
+    gap: 1rem; // sass-lint:disable-line no-misspelled-properties
+    grid-template-columns: 50% auto;
+    justify-items: stretch;
+    height: 300px;
+    margin-bottom: 1rem;
```

Comment:
you can remove this. Since you are using the `flow` utility on the parent, it's already pushing the demo button down.